tools/webdriver/webdriver/client.py

```diff
@@ -752,6 +758,60 @@ def execute_async_script(self, script, args=None):
     def screenshot(self):
         return self.send_session_command("GET", "screenshot")
 
+class BidiSession(Session):
+    def __init__(self,
+                 host,
+                 port,
+                 url_prefix="/",
+                 capabilities=None,
+                 extension=None):
+        """
+        Add a capability of "webSocketUrl": True to enable
+        Bidirectional connection in session creation.
+        """
+        self.websocket_transport = None
+        capabilities = self.__enable_websocket(capabilities)
+        super().__init__(host, port, url_prefix, capabilities, extension)
+
+    def __enable_websocket(self, caps):
```

Comment:
I think we genrally stick to single underscore names to avoid the name mangling behaviour.

Reply:
Done.